Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst

```diff
@@ -0,0 +1 @@
+Add missing 'NoReturn' to __all__ in typing.py
```

Comment:
I think both `NoReturn` and `__all__` should be in double backticks, like this ` ``__all__`` `.